<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeerClientUIClass</class>
 <widget class="QMainWindow" name="PeerClientUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PeerClientUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="peerclientui.qrc">
    <normaloff>:/PeerClientUI/Resources/wlm.ico</normaloff>:/PeerClientUI/Resources/wlm.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>36</width>
    <height>36</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>600</width>
      <height>420</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="server_lineEdit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>190</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="port_lineEdit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>240</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="server_label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>server:</string>
      </property>
     </widget>
     <widget class="QLabel" name="port_label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>240</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>port:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ConnectButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QListView" name="peerlistView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>511</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="talkButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Talk</string>
      </property>
     </widget>
     <widget class="QPushButton" name="disconnectButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>70</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Disconnect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearlogButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Clearlog</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="logtextEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>216</y>
        <width>511</width>
        <height>201</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="peerclientui.qrc"/>
 </resources>
 <connections/>
</ui>
